site/content/component/hono-server.md

```diff
@@ -44,6 +44,10 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_SERVER_REGISTRATION_ASSERTION_TOKEN_EXPIRATION`<br>`--hono.server.registrationAssertion.tokenExpiration` | no | `10` | The expiration period to use for the tokens asserting the registration status of devices. |
 | `HONO_SERVER_TRUST_STORE_PASSWORD`<br>`--hono.server.trustStorePassword` | no | - | The password required to read the contents of the trust store. |
 | `HONO_SERVER_TRUST_STORE_PATH`<br>`--hono.server.trustStorePath` | no  | - | The absolute path to the Java key store containing the CA certificates the Hono server uses for authenticating clients. The key store format can be either `JKS`, `PKCS12` or `PEM` indicated by a `.jks`, `.p12` or `.pem` file suffix respectively. |
+| `HONO_METRIC_REPORTER_GRAPHITE_ACTIVE`<br>`--hono.metric.reporter.graphite.active` | no  | `false` | Activates the metrics reporter to Graphite (or a graphite compatible system - we use InfluxDB in the `example`). |
+| `HONO_METRIC_REPORTER_GRAPHITE_HOST`<br>`--hono.metric.reporter.graphite.host` | no  | `localhost` | Sets the host, to which the metrics will be reported. |
+| `HONO_METRIC_REPORTER_GRAPHITE_PORT`<br>`--hono.metric.reporter.graphite.host` | no  | `2003` | Sets the port - 2003 ist standard for Graphite. |
+| `HONO_METRIC_REPORTER_GRAPHITE_PERIOD`<br>`--hono.metric.reporter.graphite.period` | no  | `5000` | Sets the time interval for reporting. |
```

Comment:
I wonder if it would make sense to have a separate table/block/section for describing the metrics concepts and how to configure it. In the end, all of this is not specific to Hono server but applies to all components of Hono. The only specific thing is, which metrics actually are reported. WDYT?